stories/components/AssetItem.stories.js

```diff
@@ -12,13 +12,15 @@ storiesOf('AssetItem')
     <div className='story'>
       <h2>Default</h2>
       <AssetItem
+        deleteAssetItem={console.log}
         name={ethereum.name}
         symbol={ethereum.symbol}
         address={ethereum.address}
         balance={100325.578}
       />
```

Comment:
`deleteAssetItem` prop of `AssetItem` :(
why don't we have all props like this, e.g.
nameAssetItem, symbolAssetItem etc
(sarcasm)